src/components/dev-hub/card.js

```diff
@@ -62,7 +62,7 @@ const Wrapper = styled('div')`
     transition: background-color ${animationSpeed.medium};
     width: ${({ width = 'auto' }) => width};
     ${({ highlight }) => highlight && `background: rgba(255, 255, 255, 0.3);`};
-    ${({ isclickable }) => isclickable && hoverStyles};
+    ${({ isClickable, theme }) => isClickable && hoverStyles(theme)}
```

Comment:
```suggestion
    ${({ isclickable, theme }) => isclickable && hoverStyles(theme)}
```

I think this got lost in the merge but this should be lowercase now (#415 ). This is probably why the card hover shows green text instead of the grey background it had before.